src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -165,7 +160,10 @@ public void drive(double forward, double strafe, Rotation2d rotation, boolean fi
 			moduleStates =
 					getModuleStates(
 							ChassisSpeeds.fromFieldRelativeSpeeds(
-									forward, -strafe, rotation.getRadians() * 100, getGyroRotation2d()));
+									forward,
+									-strafe,
+									rotation.getRadians() * maxSteeringSpeed * 10,
```

Comment:
Can `maxSteeringSpeed` be multiplied by 10? (So this would become `rotation.getRadians() * maxSteeringSpeed,`).

I also wonder, since `maxSteeringSpeed` is only being used in teleop, and we already have controls for adjusting the turning speed in `DriveCommand`, if we could move `maxSteeringSpeed` to `DriveCommand` and combine the two, since it's effectively "max teleop steering speed" right now.

Reply:
Fixed in commit c24d3fc